constrain/api/data_processing.py

```diff
@@ -44,13 +43,13 @@ def __init__(
         # check if data file exists
         if os.path.isfile(data_path):
             try:
-                if data_source == "EnergyPlus":
+                if data_source.lower() == "energyplus":
                     # Use CSVReader to parse EnergyPlus timestamps
                     data = CSVReader(csv_file=data_path).getseries()
                     data = DateTimeEP(data, 2000).transform()
                     data.drop("Date/Time", inplace=True, axis=1)
 
-                elif data_source == "Other":
+                elif data_source.lower() == "bms":
```

Comment:
"Other" seemed to synonymous with an "else" statement, the intent here is to deal with data from BMS.